packages/admin/cms-admin/src/contentScope/ContentScopeIndicator.tsx

```diff
@@ -1,71 +1,76 @@
-import { css } from "@mui/material";
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { SvgIconProps, useTheme } from "@mui/material";
 import { styled } from "@mui/material/styles";
 import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const bgGrey = "#5A697E";
 
 interface ContentScopeIndicatorProps {
-    variant?: "default" | "toolbar";
     global?: boolean;
+    scopes?: React.ReactNode[];
 }
 
-export const ContentScopeIndicator: React.FunctionComponent<ContentScopeIndicatorProps> = ({ children, variant = "default", global = false }) => {
+export const ContentScopeIndicator = ({ global = false, scopes = [] }: ContentScopeIndicatorProps) => {
+    const theme = useTheme();
+
+    const scopeNodes: React.ReactNode[] = [];
+
+    scopes.forEach((scope, idx, arr) => {
+        const key = scope?.toString();
+        scopeNodes.push(<span key={key}>{scope}</span>);
+
+        if (idx < arr.length - 1) {
+            scopeNodes.push(<span key={`${key}-/`}>/</span>);
+        }
+    });
+
     return (
-        <ScopeIndicator global={global} variant={variant}>
-            <Content global={global}>{children}</Content>
-            {variant === "toolbar" && <ToolbarIndicator global={global} />}
-        </ScopeIndicator>
+        <Wrapper>
+            <ScopeIndicator global={global}>
+                <DomainIcon />
+                {global ? <FormattedMessage {...messages.globalContentScope} /> : scopeNodes}
```

Comment:
When using the `global`-Prop, `scopeNodes` is still calculated, but never used. You could move the global handling into the scopeParts calculation to improve this

Reply:
Done